packages/story-editor/src/components/library/panes/text/karma/textSets.cuj.karma.js

```diff
@@ -249,10 +248,12 @@ describe('CUJ: Text Sets (Text and Shape Combinations): Using Text Sets', () =>
       // First hover text set to trigger image generation
       await fixture.events.mouse.moveRel(textSets[1], 10, 10);
 
-      await fixture.events.sleep(800);
+      await fixture.events.sleep(1500);
```

Comment:
This is such a long sleep! is it really needed? If it is, a note why might be useful to our future selves.

Reply:
Ditto https://github.com/GoogleForCreators/web-stories-wp/pull/10261/#discussion_r801057156